sqlpage/templates/form.handlebars

```diff
@@ -3,80 +3,103 @@
-                        {{~#if (or (eq autocomplete true) (eq autocomplete 1))}}autocomplete="on" {{/if~}}
-                        {{~#if (gt (len autocomplete) 1)}}autocomplete="{{autocomplete}}" {{/if~}}
-                        {{~#if autofocus}}autofocus {{/if~}}
-                    >
-                    {{/if}}{{/if}}
+                            </select>
+                        {{else}} 
+                            <input name="{{name}}" class="form-control" 
+                                {{~#if type}} type="{{type}}" {{/if~}}
+                                {{~#if placeholder}} placeholder="{{placeholder}}" {{/if~}}
+                                {{~#if value}} value="{{value}}" {{/if~}}
+                                {{~#if max}} max="{{max}}" {{/if~}}
+                                {{~#if min}} min="{{min}}" {{/if~}}
+                                {{~#if step}} step="{{step}}" {{/if~}}
+                                {{~#if minlength}} minlength="{{minlength}}" {{/if~}}
+                                {{~#if maxlength}} maxlength="{{maxlength}}" {{/if~}}
+                                {{~#if pattern}} pattern="{{pattern}}" {{/if~}}
+                                {{~#if required}} required="required" {{/if~}}
+                                {{~#if (or (eq autocomplete false) (eq autocomplete 0))}}autocomplete="off" {{/if~}}
+                                {{~#if (or (eq autocomplete true) (eq autocomplete 1))}}autocomplete="on" {{/if~}}
+                                {{~#if (gt (len autocomplete) 1)}}autocomplete="{{autocomplete}}" {{/if~}}
+                                {{~#if autofocus}}autofocus {{/if~}}  
+                            >
+                        {{/if}}
+                    {{/if}}
                     {{#if description}}
                         <small class="form-hint mt-0">{{description}}</small>
                     {{/if}}
                 </label>
             {{/if}}
-
+            </div>
         {{/each_row}}
-        <input class="btn btn-primary" type="submit" {{#if validate}}value="{{validate}}"{{/if}}>
+        </div>
+        {{#if validate}} 
+            <input class="btn
+            {{#if validate_color}} btn-{{validate_color}} {{/if}}
+            {{#if validate_shape}} btn-{{validate_shape}} {{/if}}
+            {{#if validate_outline}} btn-outline-{{validate_outline}} {{/if}}
+            {{#if validate_size}} btn-{{validate_size}} {{/if}}"
+            type="submit"
+            value="{{validate}}">
+        {{/if}}
+        {{#if reset}}
+            <input class="btn 
+            {{#if reset_color}} btn-{{reset_color}} {{/if}}
+            {{#if reset_shape}} btn-{{reset_shape}} {{/if}}
+            {{#if reset_outline}} btn-outline-{{reset_outline}} {{/if}}
+            {{#if reset_size}} btn-{{reset_size}} {{/if}}"
+            type="reset" 
+            value="{{reset}}">
+        {{/if}}
```

Comment:
Don't you think we should wrap that in some sort of container, maybe a flexbox ? Or just add a margin ? 
The way it is now, the two buttons feel too close to each other.